platform/fabric/core/generic/msp/cache.go

```diff
@@ -35,7 +35,6 @@ func NewIdentityCache(backed IdentityCacheBackendFunc, size int) *IdentityCache
 		timeout: time.Millisecond * 100,
 	}
 	go ci.run()
-
```

Comment:
do we need to remove this file?

Reply:
you mean this line I guess... don't think so ;)